### PR TITLE
Fixing the documentation

### PR DESCRIPTION
**Pull request by PhiLhoSoft**
I am myself not a native English speaker (I am French), but I noticed a number of sentences looking odd, mostly because of missing 'a' and 'the'.

I tried and fixed them, hoping not to miss something... :smile: 
HTH.

### CODE REVIEW

**Comment by pozadi** [comment]
Thanks, you did a great job! I have a couple of notes though, will add comments in the diff.

File: docs-src/descriptions/current-in-streams.jade [hunk 1]
```diff
@@ -31,21 +31,21 @@ pre(title='example')
...
-  Second issue is that it may be considered as not semantically correct behaviour
-  i.e., the fact that moments when stream emits values depend on when it gets subscribers ...
+  Second issue is that it may be considered as a not semantically correct behaviour,
+  i.e., because the moment when the stream emits values depends on when it gets subscribers...
 
 p.
-  To benefits, first — it allows you to define current value in
-  #[a(href='#from-binder') fromBinder], it's easier to show in example:
+  First benefit — it allows you to define the current value in
+  #[a(href='#from-binder') fromBinder]. It is easier as shown in this example:
```
Comment (pozadi) [inline docs-src/descriptions/current-in-streams.jade]: In original sentence I meant that it harder to explain than to show in a example, but now it has different meaning.

File: docs-src/descriptions/multiple-sources.jade [hunk 2]
```diff
@@ -82,18 +82,18 @@ p.
...
+  You can also provide a #[b combinator] function. In this case, instead
+  of emitting an array of values, they will be passed to #[b combinator] as arguments,
+  and the returned value will be emitted (same as in #[a(href="#combine") combine],
   #[a(href="#sampled-by") sampledBy], etc.)
 
 p.
   Also in #[b zip] you can pass ordinary arrays along with observables
-  in #[b sources], e.g. #[tt Kefir.zip([obs,&nbsp;[1,&nbsp;2,&nbsp;3]],&nbsp;fn)].
+  in the #[b sources], e.g. #[tt Kefir.zip([obs, [1, 2, 3]], fn)].
   In other words, #[b sources] is an array of observables and arrays,
```
Comment (pozadi) [inline docs-src/descriptions/multiple-sources.jade]: `&nbsp;` here are intentional, because code looks ugly when splitted to 2 lines.

File: docs-src/descriptions/two-sources.jade [hunk 3]
```diff
@@ -36,15 +36,15 @@ div
...
-  Returns a stream that emits latest value from #[b obs] observable
+  Returns a stream that emits the latest value from #[b obs] observable
   on each value from #[b otherObs]. Ends when #[b otherObs] ends.
 
 p.
-  You can also provide
-  #[b combinator] function which will be used to form the value emited
-  by result stream. It is called with latest values
-  from #[b obs] and #[b otherObs] as argumets. The default #[b combinator]
-  function is #[tt function(a,&nbsp;b)&nbsp;{return&nbsp;a}].
```
Comment (pozadi) [inline docs-src/descriptions/two-sources.jade]: also intentional `&nbsp;`

**Comment by pozadi** [comment]
Finally read it all through. All good except few comments above.

**Comment by PhiLhoSoft** [comment]
Suggestion: instead of using &amp;nbsp; in these code snippets, add `whitespace: pre;` to the `tt` style in your stylesheet. All your code snippets will benefit from it.

I understand the remark on the first comment, the sentence should then be "It is easier to show with an example", I think.

Do I need to update my pull request?

**Comment by pozadi** [comment]
I am just not sure all inline code snippets will be short enough to disable wrapping in all of them.

 > Do I need to update my pull request?
Would be great!

**Comment by PhiLhoSoft** [comment]
OK, updated.
I enjoyed the documentation, as it is clear and thorough.
I am eager to start playing with Kefir... :smile:

**Comment by pozadi** [comment]
> I enjoyed the documentation, as it is clear and thorough.

Thanks, and you improved it really well!

**Comment by AgentME** [comment]
The `white-space: pre-wrap;` css rule preserves sequences of whitespace without disabling wrapping.

**Comment by pozadi** [comment]
We was talking about inline snippets like this image
